Documentation~/Cinemachine3rdPersonFollow.md

```diff
@@ -0,0 +1,75 @@
+# 3rd person follow
+
+Use Cinemachine Virtual Camera’s 3rd Person Follow to keep the camera at a constant position and distance relative to the Follow target (subject to damping controls), tracking the target’s movement and rotation.
+
+The virtual camera is bolted to a virtual rig that is attached to the target. The red rods and dots visible around the target in the scene view reflect the rig setup as well as the camera position and distance relative to the target.
+
+The rig is defined by three points: the **Shoulder Offset**, the **Vertical Arm Length**, and the **Camera Distance**.
+
+Thus, the camera position is completely controlled by the Follow target’s position and orientation, and its orientation will always match that of the Follow target.
+
+
+
+For example, a rig set up with these values:
+
+![](images/CinemachineRigInspectorValuesExample.png)
+
+
+
+Produces this rig in the Scene view:
+
+![](images/CinemachineRIgSceneView.png)
+
+A virtual rig in the Scene view, using the values in the Inspector screenshot above.
+
+A: Follow target origin point
+
+B: Shoulder Offset point
+
+C: Hand offset point
+
+- The rig pivots horizontally around the Follow target’s origin point (A), and vertically around the **Shoulder Offset** point (B).
+- An arm extends vertically from the **Shoulder Offset** point, at the end of which is another point, called the hand offset point (C).
+- The **Camera Distance** point defines how far from the hand offset point to position the camera, in the direction of the Follow target’s local back vector. The camera is oriented to match the Follow target’s forward vector. The camera always looks directly at the hand.
+
+Note: C rotates around B. B rotates around A.
+
+
+
+Which results in this Game view:
+
+
+
+![](images/CinemachineRigGameViewExample.png)
+
+Note: In this example, the camera is 2 meters behind the hand.
+
+This rig can be used effectively, with a suitable shoulder offset, to produce a “third-person” camera, where the character is offset in the frame and the camera looks over the character’s shoulder. With different settings, it can be used for a 1st-person style camera.
+
+
+
+##Controlling the Camera
```

Comment:
malformed header - you need a space after the ##
Same goes for all the other headers in this doc

Reply:
Done.